installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -590,11 +580,13 @@ protected void deleteSaasBoostInstallation() {
         // needed to delete stacks via CloudFormation. delete these last.
         // TODO move these parameters to CloudFormation
         try {
-            ssm.deleteParameters(request -> request.names(
-                    "/saas-boost/" + this.envName + "/ACTIVE_DIRECTORY_PASSWORD",
-                    "/saas-boost/" + this.envName + "/METRICS_ANALYTICS_DEPLOYED",
-                    "/saas-boost/" + this.envName + "/REDSHIFT_MASTER_PASSWORD"
-            ));
+            List<String> parameterNamesToDelete = ssm.describeParameters(
+                request -> request.parameterFilters(ParameterStringFilter.builder()
+                    .key("Path")
+                    .values("/saas-boost/" + this.envName + "/")
+                    .build()))
+                .parameters().stream().map(meta -> meta.name()).collect(Collectors.toList());
+            ssm.deleteParameters(request -> request.names(parameterNamesToDelete));
```

Comment:
Remember that SSM will only let you delete 10 parameters in a single call, so you may have to batch this.